configurator/configurator-impl/pom.xml

```diff
@@ -60,6 +65,10 @@
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Bundle-Name>${project.name}</Bundle-Name>
+                        <Import-Package>
+                            !org.codice.ddf.admin.configurator.impl.*,
```

Comment:
That is an excellent question and I probably should have put a comment there because I've already forgotten why I needed to explicitly exclude the export package. I'll test it out and then add a comment.